index.bs

```diff
@@ -304,13 +304,34 @@ Report long tasks {#report-long-tasks}
-                1. Set |attribution|'s {{containerSrc}} attribute to the value of |iframe|'s <{iframe/src}> content attribute, or <code>null</code> if the attribute is absent.
-
-                    NOTE: it is intentional that we record the frame's <{iframe/src}> attribute here, and not its current URL, as this is meant primarily to help identify frames, and allowing discovery of the current URL of a cross-origin iframe is a security problem.
-                1. Set |attribution|'s {{containerId}} attribute to the value of |iframe|'s [=Element/ID=], or <code>null</code> if the ID is unset.
+            1. Set |attribution|'s {{TaskAttributionTiming/containerType}} to "<code>window</code>".
+            1. Set |attribution|'s {{TaskAttributionTiming/containerName}}, {{TaskAttributionTiming/containerSrc}}, and {{TaskAttributionTiming/containerId}} to the empty string.
+            1. If |culpritSettings| is not <code>null</code>:
+                1. Let |container| be the |culpritSettings|'s <a>responsible browsing context</a>'s <a>browsing context container</a>.
+                1. If |container| is not <code>null</code>, set |attribution|'s {{containerId}} attribute to the value of |container|'s [=Element/ID=], or the empty string if the ID is unset.
+                1. If |container| is an <{iframe}> element:
+                    1. Set |attribution|'s {{containerType}} attribute to "<code>iframe</code>".
+                    1. Set |attribution|'s {{containerName}} attribute to the value of  |container|'s <{iframe/name}> content attribute, or the empty string if the attribute is absent.
+                    1. Set |attribution|'s {{containerSrc}} attribute to the value of |container|'s <{iframe/src}> content attribute, or the empty string if the attribute is absent.
+
+                        NOTE: it is intentional that we record the iframe's <{iframe/src}> attribute here, and not its current URL, as this is meant primarily to help identify iframes, and allowing discovery of the current URL of a cross-origin iframe is a security problem.
+                1. If |container| is a <{frame}> element:
+                    <pre class=anchors>
+                    urlPrefix: https://html.spec.whatwg.org/multipage/obsolete.html; type: attribute; for: Frame
+                        text: name; url: #dom-frame-name
+                        text: src; url: #dom-frame-src
+                    </pre>
+                    1. Set |attribution|'s {{containerType}} attribute to "<code>frame</code>".
+                    1. Set |attribution|'s {{containerName}} attribute to the value of  |container|'s  <{frame/name}> content attribute, or the empty string if the attribute is absent.
+                    1. Set |attribution|'s {{containerSrc}} attribute to the value of |container|'s <{frame/src}>  content attribute, or the empty string if the attribute is absent.
```

Comment:
I wasn't able to get this to link correctly, with the correct formatting. @domenic - any ideas?

Reply:
It looks like obsolete elements are intentionally (?) missing linkable `<dfn>`s for their attributes. The HTML spec itself just using `<code>src</code>`, see e.g. https://html.spec.whatwg.org/#frames "process frame attributes". So yeah, just doing that here is probably the way to go.